ner/pom.xml

```diff
@@ -31,26 +31,32 @@
             <artifactId>illinois-core-utilities</artifactId>
             <version>3.1.10</version>
         </dependency>
-        
+        <dependency>
+            <groupId>org.cogcomp</groupId>
+            <artifactId>cogcomp-datastore</artifactId>
+            <version>1.9.6</version>
+        </dependency>
+
+        <!--Daniel: the resources commented out because we load them programmatically now. -->
```

Comment:
As mentioned above, just tried to retain easy way back. I can drop the comments.